<?xml version="1.0" encoding="UTF-8"?>
<testng-results skipped="0" failed="2" ignored="0" total="10" passed="8">
  <reporter-output>
  </reporter-output>
  <suite name="Surefire suite" duration-ms="63497" started-at="2018-10-28T17:53:47Z" finished-at="2018-10-28T17:54:50Z">
    <groups>
    </groups>
    <test name="Surefire test" duration-ms="63497" started-at="2018-10-28T17:53:47Z" finished-at="2018-10-28T17:54:50Z">
      <class name="com.qa.jive.JiveAssignment.CalculatorPageTest">
        <test-method status="PASS" signature="setup()[pri:0, instance:com.qa.jive.JiveAssignment.CalculatorPageTest@4de8b406]" name="setup" is-config="true" duration-ms="23288" started-at="2018-10-28T17:53:47Z" finished-at="2018-10-28T17:54:10Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- setup -->
        <test-method status="PASS" signature="refresh()[pri:0, instance:com.qa.jive.JiveAssignment.CalculatorPageTest@4de8b406]" name="refresh" is-config="true" duration-ms="354" started-at="2018-10-28T17:54:10Z" finished-at="2018-10-28T17:54:10Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- refresh -->
        <test-method status="PASS" signature="addTwoNumbers()[pri:1, instance:com.qa.jive.JiveAssignment.CalculatorPageTest@4de8b406]" name="addTwoNumbers" duration-ms="7445" started-at="2018-10-28T17:54:10Z" description="Add two numbers" finished-at="2018-10-28T17:54:18Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- addTwoNumbers -->
        <test-method status="PASS" signature="refresh()[pri:0, instance:com.qa.jive.JiveAssignment.CalculatorPageTest@4de8b406]" name="refresh" is-config="true" duration-ms="175" started-at="2018-10-28T17:54:18Z" finished-at="2018-10-28T17:54:18Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- refresh -->
        <test-method status="PASS" signature="subtractTwoNumbers()[pri:2, instance:com.qa.jive.JiveAssignment.CalculatorPageTest@4de8b406]" name="subtractTwoNumbers" duration-ms="5606" started-at="2018-10-28T17:54:18Z" description="Subtract two numbers" finished-at="2018-10-28T17:54:24Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- subtractTwoNumbers -->
        <test-method status="PASS" signature="refresh()[pri:0, instance:com.qa.jive.JiveAssignment.CalculatorPageTest@4de8b406]" name="refresh" is-config="true" duration-ms="156" started-at="2018-10-28T17:54:24Z" finished-at="2018-10-28T17:54:24Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- refresh -->
        <test-method status="FAIL" signature="multiplyTwoNumbers()[pri:3, instance:com.qa.jive.JiveAssignment.CalculatorPageTest@4de8b406]" name="multiplyTwoNumbers" duration-ms="5638" started-at="2018-10-28T17:54:24Z" description="Multiply two numbers" finished-at="2018-10-28T17:54:29Z">
          <exception class="java.lang.AssertionError">
            <message>
              <![CDATA[expected [19781] but found [15131]]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.AssertionError: expected [19781] but found [15131]
at org.testng.Assert.fail(Assert.java:96)
at org.testng.Assert.failNotEquals(Assert.java:776)
at org.testng.Assert.assertEqualsImpl(Assert.java:137)
at org.testng.Assert.assertEquals(Assert.java:118)
at org.testng.Assert.assertEquals(Assert.java:652)
at org.testng.Assert.assertEquals(Assert.java:662)
at com.qa.jive.JiveAssignment.CalculatorPageTest.multiplyTwoNumbers(CalculatorPageTest.java:48)
at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.lang.reflect.Method.invoke(Method.java:498)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:124)
at org.testng.internal.Invoker.invokeMethod(Invoker.java:583)
at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:719)
at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:989)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:125)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:109)
at org.testng.TestRunner.privateRun(TestRunner.java:648)
at org.testng.TestRunner.run(TestRunner.java:505)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:455)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:450)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:415)
at org.testng.SuiteRunner.run(SuiteRunner.java:364)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:84)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1208)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1137)
at org.testng.TestNG.runSuites(TestNG.java:1049)
at org.testng.TestNG.run(TestNG.java:1017)
at org.apache.maven.surefire.testng.TestNGExecutor.run(TestNGExecutor.java:135)
at org.apache.maven.surefire.testng.TestNGDirectoryTestSuite.executeMulti(TestNGDirectoryTestSuite.java:193)
at org.apache.maven.surefire.testng.TestNGDirectoryTestSuite.execute(TestNGDirectoryTestSuite.java:94)
at org.apache.maven.surefire.testng.TestNGProvider.invoke(TestNGProvider.java:146)
at org.apache.maven.surefire.booter.ForkedBooter.invokeProviderInSameClassLoader(ForkedBooter.java:383)
at org.apache.maven.surefire.booter.ForkedBooter.runSuitesInProcess(ForkedBooter.java:344)
at org.apache.maven.surefire.booter.ForkedBooter.execute(ForkedBooter.java:125)
at org.apache.maven.surefire.booter.ForkedBooter.main(ForkedBooter.java:417)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.AssertionError -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- multiplyTwoNumbers -->
        <test-method status="PASS" signature="refresh()[pri:0, instance:com.qa.jive.JiveAssignment.CalculatorPageTest@4de8b406]" name="refresh" is-config="true" duration-ms="131" started-at="2018-10-28T17:54:29Z" finished-at="2018-10-28T17:54:30Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- refresh -->
        <test-method status="PASS" signature="divideTwoNumbers()[pri:4, instance:com.qa.jive.JiveAssignment.CalculatorPageTest@4de8b406]" name="divideTwoNumbers" duration-ms="4625" started-at="2018-10-28T17:54:30Z" description="Divide two numbers" finished-at="2018-10-28T17:54:34Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- divideTwoNumbers -->
        <test-method status="PASS" signature="refresh()[pri:0, instance:com.qa.jive.JiveAssignment.CalculatorPageTest@4de8b406]" name="refresh" is-config="true" duration-ms="160" started-at="2018-10-28T17:54:34Z" finished-at="2018-10-28T17:54:34Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- refresh -->
        <test-method status="FAIL" signature="divideTwoNumbers2()[pri:5, instance:com.qa.jive.JiveAssignment.CalculatorPageTest@4de8b406]" name="divideTwoNumbers2" duration-ms="3425" started-at="2018-10-28T17:54:34Z" description="Divide another set of two numbers" finished-at="2018-10-28T17:54:38Z">
          <exception class="java.lang.AssertionError">
            <message>
              <![CDATA[expected [1] but found [199]]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.AssertionError: expected [1] but found [199]
at org.testng.Assert.fail(Assert.java:96)
at org.testng.Assert.failNotEquals(Assert.java:776)
at org.testng.Assert.assertEqualsImpl(Assert.java:137)
at org.testng.Assert.assertEquals(Assert.java:118)
at org.testng.Assert.assertEquals(Assert.java:652)
at org.testng.Assert.assertEquals(Assert.java:662)
at com.qa.jive.JiveAssignment.CalculatorPageTest.divideTwoNumbers2(CalculatorPageTest.java:62)
at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.lang.reflect.Method.invoke(Method.java:498)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:124)
at org.testng.internal.Invoker.invokeMethod(Invoker.java:583)
at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:719)
at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:989)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:125)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:109)
at org.testng.TestRunner.privateRun(TestRunner.java:648)
at org.testng.TestRunner.run(TestRunner.java:505)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:455)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:450)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:415)
at org.testng.SuiteRunner.run(SuiteRunner.java:364)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:84)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1208)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1137)
at org.testng.TestNG.runSuites(TestNG.java:1049)
at org.testng.TestNG.run(TestNG.java:1017)
at org.apache.maven.surefire.testng.TestNGExecutor.run(TestNGExecutor.java:135)
at org.apache.maven.surefire.testng.TestNGDirectoryTestSuite.executeMulti(TestNGDirectoryTestSuite.java:193)
at org.apache.maven.surefire.testng.TestNGDirectoryTestSuite.execute(TestNGDirectoryTestSuite.java:94)
at org.apache.maven.surefire.testng.TestNGProvider.invoke(TestNGProvider.java:146)
at org.apache.maven.surefire.booter.ForkedBooter.invokeProviderInSameClassLoader(ForkedBooter.java:383)
at org.apache.maven.surefire.booter.ForkedBooter.runSuitesInProcess(ForkedBooter.java:344)
at org.apache.maven.surefire.booter.ForkedBooter.execute(ForkedBooter.java:125)
at org.apache.maven.surefire.booter.ForkedBooter.main(ForkedBooter.java:417)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.AssertionError -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- divideTwoNumbers2 -->
        <test-method status="PASS" signature="refresh()[pri:0, instance:com.qa.jive.JiveAssignment.CalculatorPageTest@4de8b406]" name="refresh" is-config="true" duration-ms="175" started-at="2018-10-28T17:54:38Z" finished-at="2018-10-28T17:54:38Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- refresh -->
        <test-method status="PASS" signature="backspaceAndAddNumbers()[pri:6, instance:com.qa.jive.JiveAssignment.CalculatorPageTest@4de8b406]" name="backspaceAndAddNumbers" duration-ms="4801" started-at="2018-10-28T17:54:38Z" description="Backspace and continue to add" finished-at="2018-10-28T17:54:43Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- backspaceAndAddNumbers -->
        <test-method status="PASS" signature="refresh()[pri:0, instance:com.qa.jive.JiveAssignment.CalculatorPageTest@4de8b406]" name="refresh" is-config="true" duration-ms="139" started-at="2018-10-28T17:54:43Z" finished-at="2018-10-28T17:54:43Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- refresh -->
        <test-method status="PASS" signature="invalidOperation()[pri:7, instance:com.qa.jive.JiveAssignment.CalculatorPageTest@4de8b406]" name="invalidOperation" duration-ms="221" started-at="2018-10-28T17:54:43Z" description="Invalid operation results in ERR" finished-at="2018-10-28T17:54:43Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- invalidOperation -->
        <test-method status="PASS" signature="refresh()[pri:0, instance:com.qa.jive.JiveAssignment.CalculatorPageTest@4de8b406]" name="refresh" is-config="true" duration-ms="207" started-at="2018-10-28T17:54:43Z" finished-at="2018-10-28T17:54:43Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- refresh -->
        <test-method status="PASS" signature="invalidOperationInfinity()[pri:8, instance:com.qa.jive.JiveAssignment.CalculatorPageTest@4de8b406]" name="invalidOperationInfinity" duration-ms="1320" started-at="2018-10-28T17:54:43Z" description="Invalid operation results in Infinity" finished-at="2018-10-28T17:54:45Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- invalidOperationInfinity -->
        <test-method status="PASS" signature="refresh()[pri:0, instance:com.qa.jive.JiveAssignment.CalculatorPageTest@4de8b406]" name="refresh" is-config="true" duration-ms="120" started-at="2018-10-28T17:54:45Z" finished-at="2018-10-28T17:54:45Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- refresh -->
        <test-method status="PASS" signature="invalidOperationNaN()[pri:9, instance:com.qa.jive.JiveAssignment.CalculatorPageTest@4de8b406]" name="invalidOperationNaN" duration-ms="1270" started-at="2018-10-28T17:54:45Z" description="Invalid operation results in NaN" finished-at="2018-10-28T17:54:46Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- invalidOperationNaN -->
        <test-method status="PASS" signature="refresh()[pri:0, instance:com.qa.jive.JiveAssignment.CalculatorPageTest@4de8b406]" name="refresh" is-config="true" duration-ms="145" started-at="2018-10-28T17:54:46Z" finished-at="2018-10-28T17:54:46Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- refresh -->
        <test-method status="PASS" signature="complexOperation()[pri:10, instance:com.qa.jive.JiveAssignment.CalculatorPageTest@4de8b406]" name="complexOperation" duration-ms="3774" started-at="2018-10-28T17:54:46Z" description="Operator precedence" finished-at="2018-10-28T17:54:50Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- complexOperation -->
        <test-method status="PASS" signature="quit()[pri:0, instance:com.qa.jive.JiveAssignment.CalculatorPageTest@4de8b406]" name="quit" is-config="true" duration-ms="282" started-at="2018-10-28T17:54:50Z" finished-at="2018-10-28T17:54:50Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- quit -->
      </class> <!-- com.qa.jive.JiveAssignment.CalculatorPageTest -->
    </test> <!-- Surefire test -->
  </suite> <!-- Surefire suite -->
</testng-results>
